packages/server-boilerplate/src/orchestrator/api/ApiBuilder.ts

```diff
@@ -201,6 +210,33 @@ export class ApiBuilder {
     return this;
   }
 
+  /**
+   * Call the setTrustedProxies function periodically to update the trusted proxies
+   * because it's possible for the server's IP address to change while server is running
+   */
+  private startTrustedProxiesInterval = () => {
+    this.setTrustedProxies(); // Call it once immediately
+    setInterval(this.setTrustedProxies, TRUSTED_PROXIES_INTERVAL);
+  };
+
+  /**
+   * Dynamically set trusted proxy so that we can trust the IP address of the client
+   */
+  private setTrustedProxies = () => {
```

Comment:
I am aware that there is a bit of repeated code here but since it is only about 3 lines and the context is a bit different (ie. TS vs JS, functional vs OOP and Orchestrator vs Auth), I think it would actually be easier to maintain them separately. I don't feel too strongly about it though so am open to moving this into shared somewhere.